### DISCUSSION TITLE
httpx.ReadTimeout: The read operation timed out

### DISCUSSION THREAD

**Discussion by micoach-agustin** [deepgram/community#606] [General help]
Dear friends, 
I am transcribing with Python and MAC a 1 hour audio in mp3 (87Mb) with deepgram API. 

    deepgram = DeepgramClient(DEEPGRAM_API_KEY)
    options = PrerecordedOptions(
        model="nova-2", 
        language="es", 
        summarize=True,
        detect_topics=True, 
        smart_format=True, 
        punctuate=True, 
        paragraphs=True, 
    )

<img width="1220" alt="Screenshot 2024-02-14 at 12 40 29 a m" src="https://github.com/deepgram/community/assets/158813438/242497a9-0cd2-4e6e-a939-547eb55a3053"> 

 and I received the message: httpx.ReadTimeout: The read operation timed out
How can I fix it? Thanks,

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @micoach-agustin, do you have a request ID you can share from that failed request?

I notice you're using `summarize=True` and `detect_topics=True`, both of which are higher-latency features. To troubleshoot, can you run the same request, but with one or both of those parameters removed / set to False?
